src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -137,6 +137,16 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 	}
 
 	ngOnInit() {
+		/// For focusing the correct cell on tabbing, we hook into the focus_in and check if the active cell
+		/// is parent of the tabbed into element.
+		this._register(DOM.addDisposableListener(this.container.nativeElement, DOM.EventType.FOCUS_IN, () => {
+			const activeCellElement = this.container.nativeElement.querySelector(`.editor-group-container.active .notebook-cell.active`);
```

Comment:
What if we don't have an active cell? Does isAncestor handle that gracefully?

Reply:
yes, I checked and it would return false.